frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -43,7 +42,8 @@ pods:
         goal: RUNNING
         resource-set: server-resources
         cmd: >
-                ./bootstrap
+                ./bootstrap ;
+                pip install awscli
```

Comment:
The Docker image seems nicest, trying to package up a pip module sounds nightmarish. I'll create one with both `awscli` and `azure-cli`, I suppose.